tracer/test/test-applications/security/Samples.Security.AspNetCore5/Startup.cs

```diff
@@ -28,7 +34,38 @@ public void ConfigureServices(IServiceCollection services)
             {
                 DatabaseHelper.CreateAndFeedDatabase(Configuration.GetConnectionString("DefaultConnection"));
             }
+
             services.AddRazorPages();
+            var identityBuilder = services.AddIdentity<IdentityUser, IdentityRole>(
+                o =>
+                {
+                    o.Password.RequireDigit = false;
+                    o.Password.RequiredLength = 4;
+                    o.Password.RequireLowercase = false;
+                    o.Password.RequiredUniqueChars = 0;
+                    o.Password.RequireUppercase = false;
+                    o.Password.RequireNonAlphanumeric = false;
+                });
+            // sql lite provider doesnt seem to work on arm64 so use in memory store
+            var useSqlLiteDefault = RuntimeInformation.ProcessArchitecture != Architecture.Arm64;
+            var useSqlLite = Configuration.GetValue("UseSqlite", useSqlLiteDefault);
```

Comment:
Thinking about it, we don't actually care about the stores at all, right? so we _could_ use the in-memory stores in all cases if it helps simplify things 🤔

Reply:
Having the EF store for NET 7.0 allowed the integration tests to run on the most common use case with an ApplicationDbContext as you were mentioning, so it'd be nice to keep the various stores (the other sql lite ado ones for all other frameworks / platform, the in memory one for arm64 as sql lite didnt seem to work on it) 🤯